Hey David, yes we are seeing this problem in Staging with the new version of Sonic & Kewpie. I have extracted the error message and included it below.

Activated service account credentials for: [poet-staging@paidright-dev.iam.gserviceaccount.com]
panic: interface conversion: interface {} is nil, not *sql.DB

goroutine 1 [running]:
github.com/davidbanham/kewpie_go/v3.(*Kewpie).Connect(0x1162c20, 0xc00003832f, 0x8, 0xc0001cb0b0, 0x1, 0x1, 0x0, 0x0, 0xc00010aa40, 0xc00024fe40)
	/home/davidbanham/go/pkg/mod/github.com/davidbanham/kewpie_go/v3@v3.0.2/kewpie.go:67 +0x366
main.init.0()
	/home/davidbanham/go/src/github.com/paidright/sonic/main.go:44 +0xf2

--- a/go.mod
+++ b/go.mod
@@ -3,7 +3,7 @@ module github.com/paidright/sonic
 go 1.12
 
 require (
-	github.com/davidbanham/kewpie_go/v3 v3.0.2
+	github.com/davidbanham/kewpie_go/v3 v3.0.3
 	github.com/davidbanham/required_env v0.0.0-20150902120453-a84628a4c244
 	github.com/satori/go.uuid v1.2.0
 	github.com/stretchr/testify v1.4.0

--- a/main.go
+++ b/main.go
@@ -17,7 +17,6 @@ import (
 
 	kewpie "github.com/davidbanham/kewpie_go/v3"
 	"github.com/davidbanham/kewpie_go/v3/types"
-	"github.com/davidbanham/required_env"
 	"github.com/paidright/sonic/config"
 )
 
@@ -34,8 +33,6 @@ const (
 
 var queue kewpie.Kewpie
 
-var database *sql.DB
-
 func init() {
 	if len(os.Args) > 1 && os.Args[1] == "--version" {
 		fmt.Println(currentVersion)
@@ -43,19 +40,6 @@ func init() {
 	}
 
 	queue.Connect(config.KEWPIE_BACKEND, []string{config.QUEUE}, nil)
-
-	if config.KEWPIE_BACKEND == "postgres" {
-		required_env.Ensure(map[string]string{
-			"DB_URI": "",
-		})
-
-		db, err := sql.Open("postgres", os.Getenv("DB_URI"))
-		if err != nil {
-			log.Fatal(err)
-		}
-
-		database = db
-	}
 
 	fmt.Printf("INFO listening on queue: %s \n", config.QUEUE)
 }

--- a/main.go
+++ b/main.go
@@ -42,6 +42,8 @@ func init() {
 		os.Exit(0)
 	}
 
+	queue.Connect(config.KEWPIE_BACKEND, []string{config.QUEUE}, nil)
+
 	if config.KEWPIE_BACKEND == "postgres" {
 		required_env.Ensure(map[string]string{
 			"DB_URI": "",
@@ -54,8 +56,6 @@ func init() {
 
 		database = db
 	}
-
-	queue.Connect(config.KEWPIE_BACKEND, []string{config.QUEUE}, database)
 
 	fmt.Printf("INFO listening on queue: %s \n", config.QUEUE)
 }

